src/components/about-chat-sidebar/AboutChatSidebar.styles.ts

```diff
@@ -0,0 +1,57 @@
+import { TypographyVariantEnum } from '~/types'
+
+export const styles = {
+  wrapper: {
+    display: 'flex',
+    flexDirection: 'column',
+    gap: '24px',
+    p: '24px 8px',
+    maxWidth: '320px',
+    overflowY: 'scroll'
+  },
+  header: {
+    text: {
+      color: 'primary.700'
+    },
+    iconButton: {
+      p: '8px',
+      color: 'primary.800'
+    },
+    px: '24px',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'space-between'
+  },
+  chatInfo: {
+    avatar: {
+      width: '124px',
+      height: '124px'
+    },
+    description: {
```

Comment:
same here, don`t see a reason to make this styles nested

Reply:
> Same here, don`t see a reason to make this styles nested

fixed